e2e/fuzz/runner.go

```diff
@@ -96,8 +98,8 @@ func validateNodes(c *e2e.Cluster) {
 	if runningNodes, ok := validateCluster(c); ok {
 		currentHeight := c.GetMaxHeight(runningNodes)
 		expectedHeight := currentHeight + 10
-		log.Printf("Current height %v and waiting expected %v height.\n", currentHeight, expectedHeight)
-		err := c.WaitForHeight(expectedHeight, 3*time.Minute, runningNodes)
+		log.Printf("Running nodes: %v. Current height: %v and waiting expected: %v height.\n", runningNodes, currentHeight, expectedHeight)
+		err := c.WaitForHeight(expectedHeight, waitForHeightTimeInterval, runningNodes)
```

Comment:
Updating camel case notation of constant here:
```suggestion
		err := c.WaitForHeight(expectedHeight, WaitForHeightTimeInterval, runningNodes)
```